src/Ev.ServiceBus.IntegrationEvents/Subscription/ReceptionRegistrationBuilder.cs

```diff
@@ -1,26 +1,39 @@
-﻿using Ev.ServiceBus.Abstractions;
+﻿using System;
+using Ev.ServiceBus.Abstractions;
+using Microsoft.Azure.ServiceBus;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 
 namespace Ev.ServiceBus.IntegrationEvents.Subscription
 {
     public class ReceptionRegistrationBuilder
     {
+        private readonly ReceiverOptions _options;
         private readonly IServiceCollection _services;
 
         public ReceptionRegistrationBuilder(IServiceCollection services, ReceiverOptions receiverOptions)
         {
             _services = services;
-            Options = receiverOptions;
+            _options = receiverOptions;
         }
 
-        public ReceiverOptions Options { get; }
+        public void CustomizeMessageHandling(int maxConcurrentCalls = 1, TimeSpan maxAutoRenewDuration = default)
```

Comment:
It feels like a magic value to check. Maybe change it to ```TimeSpan?``` and check for null instead of ```!= default```  (I'm talking ofc about the insides of the ```ToIntegrationEventHandling```)

Reply:
good point